datagrid.R

```diff
@@ -0,0 +1,32 @@
+data = read.csv('/Users/mac/Documents/Toronto-Autotheft-Prediction-dev/data/cleaned_auto_weather_neighbor_dt.csv')
+## Add the latitudes and longitudes between which each observation is located
+## You can substitute any number of breaks you want. Or, a vector of fixed cutpoints
+## LATgrid and LONgrid are going to be factors. With ugly level names.
+data$LATgrid<-cut(data$Lat,breaks=45,include.lowest=T);
+data$LONgrid<-cut(data$Long,breaks=45,include.lowest=T);
+
+## Create a single factor that gives the lat,long of each observation. 
+data$IDgrid<-with(data,interaction(LATgrid,LONgrid));
+
+## Now, create another factor based on the above one, with shorter IDs and no empty levels
+data$IDNgrid<-factor(data$IDgrid); 
+levels(data$IDNgrid)<-seq_along(levels(data$IDNgrid));
```

Comment:
Make this part as a function called `add_geo_grid(auto_data)`, it returns a data object which has IDgrid and IDNgrid column added.